soap/src/java/com/zimbra/soap/admin/message/ReIndexRequest.java

```diff
@@ -52,26 +57,54 @@
     private final String action;
 
     /**
-     * @zm-api-field-description Specify reindexing to perform
-     * <br />
-     * Note: Only one of <b>{ids-comma-sep}</b> and <b>{types-comma-sep}</b> may be specified.
+     * @zm-api-field-description Specify mailboxes that will be re-indexed
+     */
+    @XmlElement(name=AdminConstants.E_MAILBOX, required=false)
```

Comment:
Worth checking whether this is an API breakage.  Think it may matter for JSON going from a single element to an array.

Reply:
I'm tempted to keep this as-is for now, since there needs to be follow-up work to finish ReiIndexRequest functionality anyways.